app/lib/search/search_service.dart

```diff
@@ -473,12 +508,31 @@ class TagsPredicate {
     return p;
   }
 
+  String _firstInvalidOverride(TagsPredicate other) {
```

Comment:
Can we give this a documentation string, and maybe use the terminology "conflictingTags" or something like that.

We don't allow overrides, so any override is an _invalid override_. Hence, the word "override" might not work so well.

Reply:
I was thinking that we allow overrides of negative tags, but you are right, we don't need those (if we have tri-state checkboxes for those we allow multiple states).